service/manager.go

```diff
@@ -85,6 +86,10 @@ func newManager(builder *Builder) (Manager, error) {
 		return nil, err
 	}
 
+	if err := m.configProvider.Get(config.Root).Populate(m); err != nil {
```

Comment:
No, if a user wants to have multiple services, they have to have a custom config and provide separate configs with an option: we can't have multiple services returned by a builder, because we have `name`, `owner`, `modules` etc. under `config.Root` already.